k8s/ansible/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# install system utilities
+RUN apt-get update && apt-get install -y \
```

Comment:
@alexander-semenets pin versions of 3d party tools

Reply:
this is pinned at Ubuntu revision level, there are only minor and security updates usually pushed to per release (16.04, 18.04, etc) repositories